dtbase/webapp/app/sensors/templates/sensor_form.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base_site.html" %}
+
+{% block title %} DTBase {% endblock title %}
+
+{% block content %}
+<div class="container">
+    <h3>Add New Sensor</h2>
+    <form id="sensor-form" method="POST">
+        <div class="form-group">
+            <label for="sensor_type">Sensor Type:</label>
+            <select class="form-control custom-select custom-input" id="sensor_type" name="sensor_type" onchange="updateForm()" required>
+                <option value="">-- Select a sensor type --</option>
+                {% for sensor_type in sensor_types %}
+                    <option value="{{ sensor_type.name }}">{{ sensor_type.name }}</option>
+                {% endfor %}
+            </select>
+            <label for="unique_identifier">Unique ID:</label>
```

Comment:
Currently, it seems like we can add a second identifier with the same unique id and it shows "Sensor added successfully", but I think it's actually doing the correct thing and isn't added. So only a matter of showing the right message.